src/components/Playback/Playback.tsx

```diff
@@ -75,7 +75,7 @@ export class Playback extends Component<Props, State> {
             });
             if (screenMedia) {
                 this.kalturaPlayerScreen = KalturaPlayer.setup({
-                    targetId: "player-wrap_screen" + uniqueId,
+                    targetId: "player-wrap-screen_" + uniqueId,
```

Comment:
```suggestion
                    targetId: "player-wrap_screen__" + uniqueId,
```
by BEM, you should use the underscore now. 
and since the `uniqueId` is unique, you should treat it like a special case (or state).

Reply:
done